guides/hack/11-built-in-types/35-enum-class.md

```diff
@@ -52,19 +69,21 @@ enum class Names : IHasName {
 hhvm.hack.lang.enable_enum_classes=1
 ```
 
-## Declaring a new enum class
+## Accessing enum class constants
```

Comment:
What is an enum class constant?

My assumption based on the `class` keyword is that we /also/ supported something like `const FOO = bar;` in here

Perhaps calling this `enum class values` with '(also known as enum class constants)' in the longer text would be clearer?

Reply:
Let's consider the `Random` enum class above, what I called 'constant" was `Random::X` and `Random::S`. You can call them members maybe ? To go with their type `HH\MemberOf<Random, ...>` ?